include/shell/shell_uart.h

```diff
@@ -31,6 +31,17 @@ struct shell_uart {
 		.ctx = (struct shell_uart *)&_name##_shell_uart		\
 	}
 
+/**
+ * @brief This function shall not be used directly. It provides pointer to shell
```

Comment:
brief starting from "this function shall not be used directly" seems strange. And isn;t that comment applicable only to dummy backend which is called internally by shell if null provided?

Reply:
Yes, it is applicable only for dummy backend. I will update the comment.